src/modules/esl-utils/dom/script.ts

```diff
@@ -1,23 +1,30 @@
+export interface loadScriptOptions {
+  crossorigin?: boolean | 'anonymous' | 'use-credentials' | '';
+  integrity?: string;
+  referrerpolicy?: ReferrerPolicy;
+}
+
 /**
- * Creates new async script tag by id and src
+ * Creates new script element with specified attributes
  */
-const createAsyncScript = (id: string, src: string): HTMLScriptElement => {
+const createScript = (id: string, src: string, options: loadScriptOptions): HTMLScriptElement => {
   const script = document.createElement('script');
   script.id = id;
-  script.async = true;
   script.src = src;
+  Object.entries(options).forEach(([name, value]) => value && script.setAttribute(name, value === true ? '' : value));
```

Comment:
Probably we can use setAttr from `dom/attr.ts`

Reply:
```ts
Object.entries(options).forEach(([name, value]) => setAttr(script, name, value));
```
I am not sure that it will be better because `setAttr()` is not short and simple. Also, it has additional dependency on `resolveDomTarget()`.

Reply:
If do not take into view that someone will want to use this utility without using a base element or mixin element. But this looks strange.
So I will switch it to `setAttr()`.